src/constants.js

```diff
@@ -6,5 +6,8 @@ export default {
     copy: {
```

Comment:
Yep - the frontend doesn't bother to submit a voucher to the API and displays `VOUCHER_SUBMIT_INVALID` if these criteria aren't met:
`this.voucherCode !== null && this.voucherCode.length > 0`